library/src/scala/quoted/Toolbox.scala

```diff
@@ -2,8 +2,56 @@ package scala.quoted
 
 import scala.annotation.implicitNotFound
 
-@implicitNotFound("Could not find implicit quoted.Toolbox.\n\nDefault toolbox can be instantiated with:\n  `implicit val toolbox: scala.quoted.Toolbox = dotty.tools.dotc.quoted.Toolbox.make`\n\nIf only needed once it can also be imported with:\n `import dotty.tools.dotc.quoted.Toolbox._`")
+@implicitNotFound("Could not find implicit quoted.Toolbox.\n\nDefault toolbox can be instantiated with:\n  `implicit val toolbox: scala.quoted.Toolbox = scala.quoted.Toolbox.make`\n\nIf only needed once it can also be imported with:\n `import scala.quoted.Toolbox.Default._`")
 trait Toolbox {
   def run[T](expr: Expr[T]): T
   def show[T](expr: Expr[T]): String
 }
+
+object Toolbox {
+
+  object Default {
+    // TODO remove? It may be better to only have one way to instantiate the toolbox
+    implicit def make(implicit settings: Settings): Toolbox = Toolbox.make
+  }
+
+  def make(implicit settings: Settings): Toolbox = {
+    val cl = getClass.getClassLoader
+    try {
+      val toolboxImplCls = cl.loadClass("dotty.tools.dotc.quoted.ToolboxImpl")
+      val makeMeth = toolboxImplCls.getMethod("make", classOf[Settings])
+      makeMeth.invoke(null, settings).asInstanceOf[Toolbox]
+    }
+    catch {
+      case ex: ClassNotFoundException =>
+        throw new ToolboxNotFoundException(
+          s"""Could not load the Toolbox class `${ex.getMessage}` in the classpath. Make sure that the compiler is on the classpath.""",
```

Comment:
"in the classpath" -> "from the JVM classpath"
"on the classpath" -> "on the JVM classpath"
(To distinguish it from the compiler classpath)

Reply:
Changed